src/components/WalletConnect/WalletConnectModal.js

```diff
@@ -43,6 +44,14 @@ export default () => {
             onDismiss={onDismiss}
           />
         );
+      case WalletConnectModalTypes.SIGN_ORACLE_DATA:
+        return (
+          <SignOracleDataModal
+            {...walletConnectModal}
+            onDismiss={onDismiss}
+            baseStyles={baseStyles}
```

Comment:
Should we remove it as we removed the others?

Reply:
Done, thanks!